src/main/java/org/alliancegenome/curation_api/model/entities/HighThroughputExpressionDatasetAnnotation.java

```diff
@@ -0,0 +1,60 @@
+package org.alliancegenome.curation_api.model.entities;
+
+import java.util.List;
+
+import org.alliancegenome.curation_api.model.entities.base.SubmittedObject;
+import org.alliancegenome.curation_api.view.View;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import org.hibernate.search.mapper.pojo.automaticindexing.ReindexOnUpdate;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexedEmbedded;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.IndexingDependency;
+
+import com.fasterxml.jackson.annotation.JsonView;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Index;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Indexed
+@Entity
+@Data
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@ToString(callSuper = true)
+@Schema(name = "HighThroughputExpressionDatasetAnnotation", description = "POJO that represents the HighThroughputExpressionDatasetAnnotation")
+public class HighThroughputExpressionDatasetAnnotation extends SubmittedObject {
+
+	@JsonView({ View.FieldsOnly.class })
+	private String name;
+
+	@IndexedEmbedded(includePaths = {"curie", "primaryCrossReferenceCurie", "crossReferences.referencedCurie", "curie_keyword", "primaryCrossReferenceCurie_keyword", "crossReferences.referencedCurie_keyword"})
+	@IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
+	@ManyToMany
+	@Fetch(FetchMode.JOIN)
+	@JoinTable(indexes = {
+		@Index(name = "htpdatasetannotation_reference_htpdataset_index", columnList = "highthroughputexpressiondatasetannotation_id"),
+		@Index(name = "htpdatasetannotation_reference_references_index", columnList = "references_id")
+	})
+	@JsonView({ View.FieldsAndLists.class })
+	private List<Reference> references;
+
+	@JsonView({ View.FieldsOnly.class })
+	private String realtedNote;
+
+	@JsonView({ View.FieldsOnly.class })
```

Comment:
Probably want indexing annotation here if this will eventually be displayed in the UI.  For integer fields I think you just need `@GenericField(projectable = Projectable.YES, sortable = Sortable.YES) `

Reply:
Added the above line for indexing.